src/sc_crawler/vendors/_azure.py

```
@@ -1164,8 +1168,14 @@ def inventory_zones(vendor):
     """
     items = []
     resources = _resources("Microsoft.Compute")
-    locations = [i for i in resources if i["resourceType"] == "virtualMachines"][0]
-    locations = {item["location"]: item["zones"] for item in locations["zoneMappings"]}
+    locations = [
+        i
+        for i in resources
+        if i.get("resource_type") == "virtualMachines"
+        or i.get("resourceType") == "virtualMachines"
+    ][0]
```

Ruff (0.15.18)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)